mlrun/datastore/targets.py

```diff
@@ -1697,21 +1721,9 @@ def add_writer_step(
         column_list = self._get_column_list(
             features=features, timestamp_key=timestamp_key, key_columns=key_columns
         )
-        if self.path and self.path.startswith("ds://"):
-            datastore_profile = datastore_profile_read(self.path)
-            attributes = datastore_profile.attributes()
-            brokers = attributes.pop(
-                "brokers", attributes.pop("bootstrap_servers", None)
-            )
-            topic = datastore_profile.topic
-        else:
-            attributes = copy(self.attributes)
-            brokers = attributes.pop(
-                "brokers", attributes.pop("bootstrap_servers", None)
-            )
-            topic, brokers = parse_kafka_url(self.get_target_path(), brokers)
+        path = self.get_target_path()
 
-        if not topic:
+        if not path:
```

Comment:
Isn't `path` _not_ going to be defined if we rely on a datastore profile?

Reply:
Seems like the code moved to the wrapper and this bit should have been removed from here too.

Reply:
In case of datastore the 'path' will be defined as 'ds://profilename/...'